modules/claims_api/app/workers/claims_api/vbms_updater.rb

```diff
@@ -7,18 +7,30 @@ module ClaimsApi
   class VBMSUpdater
     include Sidekiq::Worker
 
-    def perform(power_of_attorney_id, participant_id)
+    def perform(power_of_attorney_id)
       poa_form = ClaimsApi::PowerOfAttorney.find(power_of_attorney_id)
       service = BGS::Services.new(
         external_uid: poa_form.external_uid,
         external_key: poa_form.external_key
       )
-      service.corporate_update.update_poa_access(
-        participant_id: participant_id,
+
+      response = service.corporate_update.update_poa_access(
+        participant_id: poa_form.auth_headers['va_eauth_pid'],
         poa_code: poa_form.form_data.dig('serviceOrganization', 'poaCode'),
         allow_poa_access: 'y',
         allow_poa_c_add: allow_address_change?(poa_form) ? 'y' : 'n'
       )
+
+      if response[:return_code] == 'GUIE50000'
```

Comment:
Any chance we've confirmed with BGS that this is the appropriate value to look for?

Reply:
this was based off of what we're getting back from current [successful call we test against](https://github.com/department-of-veterans-affairs/bgs-ext/blob/master/spec/bgs/services/corporate_update_spec.rb#L36-L37), but I will double check with the BGS team to validate